### DISCUSSION TITLE
Interaction with other online services

### DISCUSSION THREAD

**Discussion by ClearlyClaire** [iSoron/uhabits#17] [Feature Requests]
Hi, I've just discovered your app on F-Droid, and I have to say I'm impressed with the clean and useful UI!
I'm currently using Habitica (on my own server) which is more feature-rich, although much more bloated and depending on a network connection.
As Loop Habit Tracker more or less corresponds to the “Dailies” part of Habitica, I would be interested in some sort of synchronization between the two.

**Comment by ClearlyClaire** [comment]
In order not to bloat Loop Habit Tracker, this could be done using external applications. So I guess I'm just asking for a generalization of #6.

**Comment by iSoron** [comment]
Glad you like the app. I have made your request specific about online services, since integration with other apps is already partially covered by #6, as you noted.

Integration with other websites is something quite complicated, so this feature will have to wait, I guess, until at least we have the ability to sync habit data across devices.

**Comment by Vic-r** [comment]
What about interactions with a Calendar? I use Google one and like some futures such as repeat task/ meeting / event (daily,weekly,monthly etc).
Make please an option to recognize tasks, events, meetings  etc as habits
